README.md

```diff
@@ -3,27 +3,28 @@ Repositorio para el proyecto de Infraestructura Virtual.
 
 ## Problema a Resolver
 ### Formulación del Problema 
-Un alumno de Ingeniería Informática tiene problemas de presupuesto, ya que no consigue llegar a final de mes económicamente. Cada mes tiene unos gastos fijos determinados, como alquiler, comunidad de vecinos e internet. Dentro de su presupuesto también tiene una serie de gastos variables donde se pueden encontrar gastos por ocio en función de si quiere ir al gimnasio o realizar actividades deportivas con sus compañeros de clase semanalmente y gastos de la vivienda como electricidad, agua y gas, que tienen un precio variable a lo largo del año. Por último, destina también parte de su presupuesto a ahorro. Le gustaría poder mantener correctamente un control de su presupuesto personal con el objetivo de llegar bien a final de mes, conociendo el gasto que puede destinar un determinado día. 
+Un alumno de Ingeniería Informática tiene problemas de presupuesto, ya que no consigue llegar a final de mes económicamente. Cada mes tiene unos gastos esenciales que cubrir, como alquiler, comunidad de vecinos e internet. Además, debe cubrir otros gastos que varían a lo largo del tiempo, como compras o facturas (electricidad, agua, gas). Además, destina parte de su presupuesto a ahorro. 
+
+El estudiante tiene acceso a datos financieros en formato digital, descargando su estado bancario, pero tiene dificultades en el análisis de los mismos para poder tomar decisiones informadas. Actualmente, no cuenta con una herramienta con una herramienta que le permita categorizar sus gastos o calcular cuánto puede gastar un determinado día sin comprometer sus necesidades esenciales ni sus metas de ahorro. Le gustaría una solución que le facilitase esta tarea, sin tener que analizar detalladamente cada uno de los gastos empleando demasiado tiempo en ello.  
 
 ## Desarrollo del Proyecto
 ### Historias de Usuario
 Se han definido las siguientes historias de usuario: 
-1. [HU1] - Falta de Control sobre los Gastos Fijos y Variables
+1. [HU1] - Falta de Control sobre los Gastos  
 2. [HU2] - Falta de Información para Decidir si un Gasto Adicional es Viable
 
 [Historias de Usuario](/docs/historias_usuario.md)  
 
 ### User Journeys
 Se han definido los siguientes user journeys:
-1. [UserJourney1] - Registro de Gastos Fijos
-2. [UserJourney2] - Registro y Seguimiento de Gastos Variables
-3. [UserJourney3] - Evaluación de Gasto Adicional
+1. [UserJourney1] - Registro de Gastos  
+3. [UserJourney2] - Evaluación de Gasto Adicional  
 
 [User Journeys](/docs/user_journeys.md)  
 
 ### Milestones
 Se han definido los siguientes milestones:
 1. [M0] - Modelo del Problema
-2. [M1] - Algoritmo de Gastos Adicionales
+2. [M1] - Algoritmo de Gastos 
```

Comment:
Por favor, mirad para qué sirven los corchetes en markdown y no los uséis de forma arbitraria. Esto no crea un enlace, crea una clave que más adelante en el mismo fichero tenéis que definir.